Gems/AtomLyIntegration/CommonFeatures/Code/Source/CoreLights/DirectionalLightComponentConfig.cpp

```diff
@@ -108,24 +109,30 @@ namespace AZ
             return m_isShadowmapFrustumSplitAutomatic;
         }
 
+        bool DirectionalLightComponentConfig::IsShadowDisabled() const
+        {
+            return !m_shadowEnabled;
+        }
+
         bool DirectionalLightComponentConfig::IsCascadeCorrectionDisabled() const
         {
-            return (m_cascadeCount == 1 || !m_isCascadeCorrectionEnabled);
+            return (!m_shadowEnabled || m_cascadeCount == 1 || !m_isCascadeCorrectionEnabled);
         }
 
         bool DirectionalLightComponentConfig::IsShadowFilteringDisabled() const
         {
-            return (m_shadowFilterMethod == ShadowFilterMethod::None);
+            return (!m_shadowEnabled || m_shadowFilterMethod == ShadowFilterMethod::None);
         }
 
         bool DirectionalLightComponentConfig::IsShadowPcfDisabled() const
         {
-            return !(m_shadowFilterMethod == ShadowFilterMethod::Pcf);
+            return !(!m_shadowEnabled || m_shadowFilterMethod == ShadowFilterMethod::Pcf);
```

Comment:
Minor Recommendation for clarity. 
return !m_shadowEnabled ||  m_shadowFilterMethod != ShadowFilterMethod::Pcf